src/NuGet.Clients/StandaloneUI/StandaloneUIContext.cs

```diff
@@ -27,7 +27,7 @@ internal class StandaloneUIContext : NuGetUIContextBase
             IOptionsPageActivator optionsPageActivator,
             IEnumerable<NuGetProject> projects)
             :
-                base(sourceProvider, solutionManager, packageManager, uiActionEngine, packageRestoreManager, optionsPageActivator, projects)
+                base(sourceProvider, solutionManager, packageManager, uiActionEngine, packageRestoreManager, optionsPageActivator, projects, null)
```

Comment:
1) Use parameter name when parameter value is null, true or false.
2) Break long line